.github/workflows/clients-go-build.yml

```diff
@@ -129,7 +129,7 @@ jobs:
             go-binaries-${{ github.sha }}
       - run: mkdir ~/.ssh && echo -e "${{ secrets.TIGERBEETLE_GO_DEPLOY_KEY }}" > ~/.ssh/id_ed25519 && chmod 600 ~/.ssh/id_ed25519
       - run: git clone --no-checkout git@github.com:tigerbeetledb/tigerbeetle-go.git ~/tigerbeetle-go && cp -R . ~/tigerbeetle-go
-      - run: echo -e "# tigerbeetle-go\nThis repo has been automatically generated from [tigerbeetledb/tigerbeetle@${{ github.sha }}](https://github.com/tigerbeetledb/tigerbeetle/commit/${{ github.sha }}). Please see [tigerbeetledb/tigerbeetle](https://github.com/tigerbeetledb/tigerbeetle) for documentation and contributions." > ~/tigerbeetle-go/README.md
+      - run: echo -e "# tigerbeetle-go\nThis repo has been automatically generated from [tigerbeetledb/tigerbeetle@${{ github.sha }}](https://github.com/tigerbeetledb/tigerbeetle/commit/${{ github.sha }}) to keep binary blobs out of the monorepo. Please see [tigerbeetledb/tigerbeetle/src/clients/go](https://github.com/tigerbeetledb/tigerbeetle/src/clients/go) for documentation and contributions." > ~/tigerbeetle-go/README.md
```

Comment:
The link should be https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/go

Reply:
oops, thanks. force pushed